<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nico Kruithof</author>
 <class>Periodic_2_triangulation_2</class>
 <widget class="QMainWindow" name="Periodic_2_triangulation_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL Periodic_2 Triangulation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="maximumSize">
       <size>
        <width>100000</width>
        <height>100000</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolEditTriangulationBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertPoint"/>
   <addaction name="actionMovingPoint"/>
   <addaction name="separator"/>
   <addaction name="actionShowTriangulation"/>
   <addaction name="actionShowLocate"/>
   <addaction name="separator"/>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPoints"/>
    <addaction name="actionSavePoints"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints"/>
    <addaction name="actionConvertTo1Cover"/>
    <addaction name="actionConvertTo9Cover"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInsertPoint"/>
    <addaction name="actionMovingPoint"/>
    <addaction name="actionCircumcenter"/>
    <addaction name="actionShowLocate"/>
    <addaction name="separator"/>
    <addaction name="actionShowTriangulation"/>
    <addaction name="actionShowVoronoi"/>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints">
   <property name="text">
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionMovingPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/moving_point.png</normaloff>:/cgal/Actions/icons/moving_point.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Simulate insertion</string>
   </property>
   <property name="toolTip">
    <string comment="The comment">Simulate Insertion</string>
   </property>
   <property name="statusTip">
    <string comment="and its comment">Move mouse with left button pressed to see where point would be inserted</string>
   </property>
   <property name="whatsThis">
    <string comment="what">whats this</string>
   </property>
  </action>
  <action name="actionInsertPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/inputPoint.png</normaloff>:/cgal/Actions/icons/inputPoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert Point</string>
   </property>
   <property name="toolTip">
    <string>Insert Point</string>
   </property>
   <property name="statusTip">
    <string>Left: Insert vtx</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowVoronoi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Voronoi Diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShowTriangulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/triangulation.png</normaloff>:/cgal/Actions/icons/triangulation.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Periodic_2 Triangulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCircumcenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/circumcenter.png</normaloff>:/cgal/Actions/icons/circumcenter.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Circumcenter</string>
   </property>
   <property name="toolTip">
    <string>Draw circumcenter</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/fit-page-32.png</normaloff>:/cgal/Actions/icons/fit-page-32.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionShowLocate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Periodic_2_triangulation_2.qrc">
     <normaloff>:/cgal/Actions/icons/conflict_zone.png</normaloff>:/cgal/Actions/icons/conflict_zone.png</iconset>
   </property>
   <property name="text">
    <string>Show Conflict Zone</string>
   </property>
   <property name="toolTip">
    <string>Point location</string>
   </property>
  </action>
  <action name="actionConvertTo9Cover">
   <property name="text">
    <string>Convert to 9-cover</string>
   </property>
  </action>
  <action name="actionConvertTo1Cover">
   <property name="text">
    <string>Convert to 1-cover</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Periodic_2_triangulation_2.qrc"/>
  <include location="../icons/File.qrc"/>
  <include location="../resources/CGAL.qrc"/>
  <include location="../icons/Triangulation_2.qrc"/>
  <include location="../icons/Input.qrc"/>
 </resources>
 <connections/>
</ui>
